onboarding/sessions/tour-of-artsy.md

```diff
@@ -0,0 +1,162 @@
+      - [Fairs app code](https://github.com/artsy/force/tree/master/src/desktop/apps/fairs)
+      - [Administered in Waves](https://admin-fairs-staging.artsy.net/)
+  - **Magazine**
+    - Curated by our Editorial team
+    - Reference:
+      - [Administered and modeled in Positron](https://github.com/artsy/positron)
+      - [Articles app code](https://github.com/artsy/force/tree/master/src/desktop/apps/articles)
+
+4. Browse individual entities
+  - **Artwork page**
+    - Goal of the artwork page is to give the collector information about the artwork, confidence in their buying decision
+    - Reference:
+      - [Artwork model](https://github.com/artsy/gravity/blob/master/app/models/domain/artwork.rb)
+      - [Artwork app code](https://github.com/artsy/force/tree/master/src/desktop/apps/artwork)
+  - **Artist page**
+    - Recently made this page responsive. Is a top entry-point for "shopping" for an artist's work
+    - Reference:
+      - [Artist model](https://github.com/artsy/gravity/blob/master/app/models/domain/artist.rb)
+      - [Artist app code (Force)](https://github.com/artsy/force/tree/master/src/desktop/apps/artist)
+      - [Artist app code (Reaction)](https://github.com/artsy/reaction/tree/master/src/Apps/Artist)
+  - **Auction page**
+    - Reference:
+      - [filter/sale_artworks API](https://github.com/artsy/gravity/blob/master/app/api/v1/filter_endpoint.rb)
+      - [Auction app code](https://github.com/artsy/force/tree/master/src/desktop/apps/auction)
+  - **Gallery profile page**
+    - Vanity URL since it may be shared
+    - Reference:
+      - [App code](https://github.com/artsy/force/tree/master/src/desktop/apps/partner)
+  - **Article page**
+    - Many different types of articles, determines formatting/content options
+    - Reference:
+      - [App code](https://github.com/artsy/force/tree/master/src/desktop/apps/article)
+  - **Fair microsite**
+    - Vanity URL since it will be shared
+    - Reference:
+      - [App code](https://github.com/artsy/force/tree/master/src/desktop/apps/fair)
+
+5. Follow an artist/gallery/category from the UI
+  - Notice that these now show up in your user settings (and will continue to power recs, similarity, etc.)
+  - Reference:
+    - [FollowArtist model](https://github.com/artsy/gravity/blob/master/app/models/domain/follow_artist.rb)
+
+6. Save an artwork for later
+  - Reference:
+    - [CollectedArtwork model](https://github.com/artsy/gravity/blob/master/app/models/domain/collected_artwork.rb)
+
+### Ways that you can buy art as an Artsy user...
+1. Inquire on an artwork that is owned by a known test account (i.e. Invoicing Demo Partner)
+  - Inquiries are sent to gallery contacts that are subscribed to receive inquiry messages
+  - Galleries can respond via Conversations (in CMS) or email
+  - Collectors can respond via Messaging (in the iOS app) or email
+  - Inquiries are a key metric for our teams as it can indicate an intent to buy (and eventually lead to a purchase)
```

Comment:
I think one interesting fact for new team members joining Artsy is that purchases have, historically, not been a first-class tracked thing in the Artsy domain model. While, I think, this is changing with the BNMO effort, I would recommend being explicit with this in these docs.

Reply:
Well, not _all_ purchases are tracked in Artsy. Invoices due map to purchases, IIRC, but I believe Chung-Yi mentioned in his onboarding session that many purchases happen out of Artsy.